src/js_realm.hpp

```diff
@@ -206,6 +209,7 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
         {"schemaVersion", wrap<schema_version>},
         {"clearTestState", wrap<clear_test_state>},
         {"copyBundledRealmFiles", wrap<copy_bundled_realm_files>},
+        {"_waitForDownload", wrap<wait_for_download_completion>},
```

Comment:
Do we need to use the `_`? Any reason this shouldn't be something users could call themselves?